packages/ove-asset-manager/src/OVE.Service.AssetManager/Views/OVEAssetModel/Create.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@model OVE.Service.AssetManager.Models.OVEAssetModel
+@using OVE.Service.AssetManager.Domain
+@inject ServiceRepository FileTypes
+
+@{
+    ViewData["Title"] = "Create";
+}
+
+<h2>Create</h2>
+
+<h4>ImageFileModel</h4>

```

Comment:
Please give this page a suitable name rather than ImageFileModel

Reply:
fixed